packages/@okta/i18n/src/properties/login_cs.properties

```diff
@@ -875,6 +875,7 @@ common.logo.alt = logo aria
 ## Generic
 oie.authenticator.enroll.error.fail = Ov\u011B\u0159ovac\u00ED aplikaci nelze zaregistrovat. Zkuste to znovu.
 oie.authenticator.verify.error.fail = Ov\u011B\u0159ovac\u00ED aplikaci nelze ov\u011B\u0159it. Zkuste to znovu.
+oie.configuration.error = N\u011Bco se nepovedlo. Byla zji\u0161t\u011Bna potenci\u00E1ln\u011B chybn\u00E1 konfigurace. Kontaktujte pros\u00EDm podporu.
 
```

Comment:
This is something which I did not change in login.properties. Do we still need to take this or any other team requested?

Reply:
@lesterchoi-okta

Reply:
@lokeshhemrajani-okta  Please include in your PR. There is no 1:1 PR content mapping. Language quality improvement  is an ongoing effort.